src/schedule/ll_schedule.c

```diff
@@ -223,8 +223,10 @@ static void schedule_ll_task_insert(struct task *task, struct list_item *tasks)
 	/* tasks are added into the list in order */
```

Comment:
Please explain more on what the right order means. It looks like there is a problem with tasks with the same priority.